src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -204,15 +204,15 @@ public void updateFilteredIncomeList(Predicate<Entry> predicate) {
             throw new AssertionError("This method should not be called.");
         }
 
-        @Override
-        public void updateExpensePieChart() {
-            throw new AssertionError("This method should not be called.");
-        }
-
-        @Override
-        public void updateIncomePieChart() {
-            throw new AssertionError("This method should not be called.");
-        }
+        // @Override
```

Comment:
Commented out first because this needs to wait for `ViewCommand`

Reply:
👍🏻